docs/adr/adr-013-blocksync-overhaul-part-2.md

```diff
@@ -0,0 +1,418 @@
+# ADR #011: Block Data Sync Overhaul: Part II - Network
+
+## Authors
+
+@Wondertan @renenay
+
+> This ADR took a while to get through all the necessary details. Special thanks to awesome @renenay for dozens of
+> deep dive call, note-taking, thoughts and ideas.
+> Similarly, the time was spent on the experiments with the format of the documents was refactored 3 times.
+
+## Glossary
+
+- LN - Light Node
+- FN - Full Node
+- BN - Bridge Node
+- Core Network - Consensus Network of Validators/Proposers producing new Celestia Blocks
+- [EDS(Extended Data Square)](https://github.com/celestiaorg/rsmt2d/blob/master/extendeddatasquare.go#L10) - plain Block
+  data omitting headers and other block metadata
+- ODS - Original Data Square or the first quadrant of the EDS. Contains real user data and padding
+- Recent EDS - Newly produced EDS coming from the Core consensus network
+- ND - Namespaced data/shares. Constitutes EDS
+
+## Overview
+
+This document is a continuation of the BlockSync Overhaul design. The previous document(TODO: Link) described design for the
```

Comment:
permaling for: https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-11-blocksync-overhaul.md ?